### PR TITLE
Remove default value of short for Text leading property

### PR DESCRIPTION
**Pull request by jennyscript**
## TODO

- [ ] Documentation
- [ ] Tests
- [ ] Experimental evidence (required for Masonry changes)
- [ ] Accessibility checkup

### CODE REVIEW

File: packages/gestalt/src/Text.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ export default function Text({
   smSize,
   mdSize,
   lgSize,
-  leading = 'short',
+  leading,
```
Comment (pfarejowicz) [inline packages/gestalt/src/Text.js]: Can we update the react proptypes down below as well? Noticed they are missing this property

Comment (jennyscript) [reply packages/gestalt/src/Text.js]: There is a propType specified here https://github.com/pinterest/gestalt/blob/0ee5c40a3f15a3e4c4fa4704f6f61657286c8671/packages/gestalt/src/Text.js#L150 Was there another place missing it?

Comment (pfarejowicz) [reply packages/gestalt/src/Text.js]: My mistake, the page formatting in the side by side diff view got me

**Comment by jennyscript** [comment]
The Travis CI Branch step appears to be timing out here. Since the Pull Request Travis CI step is passing, I'm going to go ahead and admin merge. This possibly needs a fix in the Travis settings to turn "Build pushed branches" off, but I don't have admin rights to Travis https://docs.travis-ci.com/user/pull-requests/#double-builds-on-pull-requests. @chrislloyd Could you check the settings?

**Pull request merged**